src/index.test.ts

```diff
@@ -1337,6 +1343,7 @@ describe('Running @erc725/erc725.js tests...', () => {
           SIGN: false,
           EXECUTE_RELAY_CALL: false,
           ERC4337_PERMISSION: false,
+          ALL_PERMISSIONS: false,
```

Comment:
It might be worth adding some extra tests to tests that when all permission is set and others are set to false, it removes them, etc... More cases

Reply:
yes ideally we can randomize them like i did for some tests i added. I find it not efficient to do one test at a time. Will think about it